cli/add.go

```diff
@@ -40,9 +41,9 @@ func validateEnv(key, value string, envList []models.EnvironmentItemModel) (stri
 	if configs.EnvBytesLimitInKB > 0 {
 		if valueSizeInBytes > configs.EnvBytesLimitInKB*1024 {
 			valueSizeInKB := ((float64)(valueSizeInBytes)) / 1024.0
-			log.Warnf("environment value (%s...) too large", value[0:100])
+			log.Warnf("`%s` environment value (%s...) too large", key, value[0:100])
```

Comment:
Please check the logging format in the whole code before you start to use a new way to highlight a detail. **`** character was never used in any of our logs.

For example:
```
log.Warnf("environment (%s) value (%s...) too large", key, value[0:100])
```